### DISCUSSION TITLE
Invalid npm package name in README?

### DISCUSSION THREAD

**Discussion by williamboman** [polarmutex/beancount-language-server#102] [Q&A]
Hello! I'm trying to install this package by using the `@bryall/beancount-langserver` npm package, as suggested in the README. This is, seemingly correctly, yielding a 404 response from npm. However, the `beancount-langserver` npm package exists, and it seems to be the correct one. Am I perhaps missing something?

Thanks!

**Comment by polarmutex** [comment]
that is the correct one, but future updates will be a rust binary
